packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -254,6 +254,7 @@ contract L2OutputOracle_Initializer is UpgradeGovernor_Initializer {
     // Test data
     address internal asserter = 0x000000000000000000000000000000000000aAaB;
     address internal challenger = 0x000000000000000000000000000000000000AAaC;
+    address internal withdrawAcc = 0x000000000000000000000000000000000000AaaD;
```

Comment:
Why not change this address to be distinct from guardian address?

Reply:
Oh, I didn't know it's the same with guardian address. I'll change it.